src/components/Header/index.jsx

```diff
@@ -3,10 +3,12 @@ import Link from "next/link";
 import Image from "next/image";
 import styles from "./header.module.css";
 import { Button, Container, Typography } from "@mui/material";
+import  TransitionAlerts from "../Alert";
 
 const Header = ({ navComponent }) => {
   return (
     <>
+      <TransitionAlerts></TransitionAlerts>
```

Comment:
Use a self closing tag

Reply:
[3908d19](https://github.com/BizTech-Morocco/mosa3ada.biztech.ma/pull/33/commits/3908d1945669a544c23b0d6a5beeb0ec9df15320)

Reply:
![image](https://github.com/BizTech-Morocco/mosa3ada.biztech.ma/assets/44750827/adb5100a-1e40-454f-aabd-90c20df80d6b)
Still getting the same bug

Reply:
If you're only utilizing the CacheProvider just for RTL support, there are alternative workarounds to get same results